morpheus/pipeline/multi_message_stage.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, c: Config):
 
         super().__init__(c)
 
-    def _post_build_single(self, seg: neo.Segment, out_pair: StreamPair) -> StreamPair:
+    def _post_build_single(self, seg: srf.Builder, out_pair: StreamPair) -> StreamPair:
```

Comment:
Might make sense to name it `builder: srf.Builder`, for readablity.

Reply:
Renaming all `seg: srf.Builder` to `builder: srf.Builder` and resolving all similar comments